cheerio-scraper/INPUT_SCHEMA.json

```diff
@@ -59,6 +59,23 @@
 			"default": {},
 			"editor": "proxy"
 		},
+		"proxyRotation": {
+			"title": "Proxy rotation",
+			"type": "string",
+			"description": "This property indicates the strategy of proxy rotation and can only be used in conjunction with Apify Proxy. The recommended setting automatically picks the best proxies from your available pool and rotates them evenly, discarding proxies that become blocked or unresponsive. If this strategy does not work for you for any reason, you may configure the scraper to either use a new proxy for each request, or to use one proxy as long as possible, until the proxy fails. IMPORTANT: This setting will only use your available Apify Proxy pool, so if you don't have enough proxies for a given task, no rotation setting will produce satisfactory results.",
+			"default": "RECOMMENDED",
+			"editor": "select",
+			"enum": [
+				"RECOMMENDED",
+				"PER-REQUEST",
```

Comment:
Bit late to the show, but... by convention, we're using underscored in the constant names, i.e. `PER_REQUEST`. BTW have you guys discussed this approach with the team and got feedback on it?

Reply:
Yes. I asked them and they liked this idea.

Reply:
Cool. But pls at least change the constants to underscore.